docs/sql-ref-syntax-aux-show-databases.md

```diff
@@ -19,4 +19,31 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+Lists the databases that match a optionally supplied string pattern. If no
+pattern is supplied then the command lists all the databases in the system.
+Please note that the usage of SCHEMAS and DATABASES are interchangable
+and mean the same thing.
+
+### Syntax
+{% highlight sql %}
+SHOW {DATABASES|SCHEMAS} [LIKE 'pattern']
+{% endhighlight %}
+
+### Parameters
+##### ***LIKE 'pattern'***:
+A string pattern that is used to match the databases. In pattern, * matches any number of characters.
```

Comment:
Back tick `*` or quote it. So this doesn't work like SQL LIKE where `%` is a wildcard?

Reply:
@srowen changed to use backtick. Actually only "*" works as a wildcard for this statement.